include/envoy/http/header_map.h

```diff
@@ -144,13 +139,20 @@ class HeaderString {
   /**
    * @return whether a substring exists in the string.
    */
-  bool find(const char* str) const { return strstr(c_str(), str); }
+  bool find(const char* str) const {
```

Comment:
Can you add a TODO to change all APIs here that could use a `string_view` to use one? This applies to `find`, removing the non-string view `setCopy` and maybe others? It would be great to track this in a TODO and then maybe do a follow up?

Reply:
Yeah - I have written a few notes about places like this I found. I'm going to create an issue that enumerates the follow ups in one place.